integration/build.gradle

```diff
@@ -30,7 +30,7 @@ dependencies {
 
         datarepoClient = "1.0.155-SNAPSHOT"
         testRunnerVersion = "0.1.3-SNAPSHOT"
-        samClient = "0.1-61135c7"
+        samClient = "0.1-33b709f-SNAP"
```

Comment:
Note: will update to non-SNAP once https://github.com/broadinstitute/sam/pull/769 is merged